spec/unit/libraries/resource_provider_spec.rb

```diff
@@ -287,6 +287,100 @@ class CatastrophicProvider < OneviewCookbook::ResourceProvider
     end
   end
 
+  describe '#add_to_scopes' do
+    let(:scope1) { OneviewSDK::API300::Synergy::Scope.new(@client, name: 'Scope1', uri: '/rest/fake/1') }
+    let(:scope2) { OneviewSDK::API300::Synergy::Scope.new(@client, name: 'Scope1', uri: '/rest/fake/2') }
+
+    before do
+      expect(res.item).to receive(:retrieve!).and_return(true)
+      allow(res.context).to receive(:scopes).and_return(['Scope1', 'Scope2'])
```

Comment:
Context won't receive the scopes, the new resource will.

Reply:
But @new_resource receives the same attributes of context, right?!
Anyway, I change it. =)